core/src/main/java/org/apache/gravitino/cache/EntityCacheKey.java

```diff
@@ -138,7 +138,7 @@ public int hashCode() {
    */
   @Override
   public String toString() {
-    String stringExpr = identifier.toString() + ":" + type.getShortName();
+    String stringExpr = identifier.toString() + ":" + type.toString();
```

Comment:
update to `type.toString` , because in the `CaffeineEntityCache` class.
when addCache, it will put data to `cacheData` (this struct store the data) and add key to `cacheIndex` struct (use EntityCache,toString as key).

So if without type prefix, there will be a edge case. like this UT (https://github.com/apache/gravitino/blob/main/core/src/test/java/org/apache/gravitino/cache/TestCaffeineEntityCache.java#L229) . cache store 2 entities, but the size return 1.

 https://github.com/apache/gravitino/blob/main/core/src/main/java/org/apache/gravitino/cache/CaffeineEntityCache.java#L295